src/index.html

```diff
@@ -1,11 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Javascript Capstone</title>
-</head>
-<body>
-    <h1>Javascript Capstone</h1>
-</body>
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://kit.fontawesome.com/6c26cc115b.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <header>
+      <img class="logo" src="https://shorturl.at/aqVZ5" alt="" />
+      <nav>
+        <ul>
+          <li>TVmaze</li>
+          <li>Series</li>
+          <li><a href="index.html">Feature</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section id="cardContainer">
+        <!-- <div>
+          <img src="" alt="" />
+          <h2></h2>
+          <button type="button"><img src="" alt="love icon" />likes</button>
+          <button type="button">Comments</button>
+        </div>
+        <div>
+          <img src="" alt="" />
+          <h2></h2>
+          <button type="button"><img src="" alt="love icon" />likes</button>
+          <button type="button">Comments</button>
+        </div>
+        <div>
+          <img src="" alt="" />
+          <h2></h2>
+          <button type="button"><img src="" alt="love icon" />likes</button>
+          <button type="button">Comments</button>
+        </div>
+        <div>
+          <img src="" alt="" />
+          <h2></h2>
+          <button type="button"><img src="" alt="love icon" />likes</button>
+          <button type="button">Comments</button>
+        </div>
+        <div>
+          <img src="" alt="" />
+          <h2></h2>
+          <button type="button"><img src="" alt="love icon" />likes</button>
+          <button type="button">Comments</button>
+        </div>
+        <div>
+          <img src="" alt="" />
+          <h2></h2>
+          <button type="button"><img src="" alt="love icon" />likes</button>
+          <button type="button">Comments</button>
+        </div> -->
+      </section>
+    </main>
+    <footer>
+      <p>Created by Microverse under CC license</p>
+    </footer>
+    <!-- <script type="module" src="index.js"></script> -->
```

Comment:
Please also delete this commented script line.

Reply:
- Deleted commented code